cli/src/cmd/forge/selectors.rs

```diff
@@ -13,6 +15,28 @@ use foundry_common::{
 /// CLI arguments for `forge selectors`.
 #[derive(Debug, Clone, Parser)]
 pub enum SelectorsSubcommands {
+    /// Check for selector collisions between contracts
+    #[clap(visible_alias = "co")]
+    Collision {
+        /// First contract
+        #[clap(
+            help = "The first of the two contracts for which to look selector collisions for, in the form `(<path>:)?<contractname>`",
+            value_name = "FIRST_CONTRACT"
+        )]
+        first_contract: ContractInfo,
+
+        /// Second contract
+        #[clap(
+            help = "The second of the two contracts for which to look selector collisions for, in the form `(<path>:)?<contractname>`",
+            value_name = "SECOND_CONTRACT"
+        )]
+        second_contract: ContractInfo,
+
+        /// Support build args
+        #[clap(flatten)]
+        build: Box<CoreBuildArgs>,
```

Comment:
is this box necessary?

Reply:
Rust won't complain without it, but clippy does as without the two elements of the SelectorsSubcommands enum differ quite a bit in size